### PR TITLE
Issue 416: Add cities visited labels in friend's profile

### LINKED ISSUE #416: Add cities visited labels in friend's profile
### Steps to go to friend's profile
- Go to `My Friends` from the navigation drawer
- Click on any friend

### Expected behaviour
Below the friend's profile photo, show the labels of the cities visited by the user.
labels should look something like GitHub label only (with city name). Assign any random color to the label.
screen shot 2018-08-08 at 12 18 48 am

### Endpoint to use
`get-trip-cities/<int:user_id>`
https://github.com/project-travel-mate/server/issues/104

### PR DESCRIPTION
**Pull request by nikhilbansal97**
## Description

Added the cities visited labels in the profile of a person

Screenshot:

screenshot_1539792653

Fixes #416 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
Tested on emulator and device.

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by nikhilbansal97** [comment]
@Swati4star I have updated the code to look according to the screenshot.

File: Android/app/src/main/java/io/github/project_travel_mate/friend/FriendCityRecyclerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    @Override
+    public FriendCityViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        return new FriendCityViewHolder(
+                LayoutInflater.from(mContext).inflate(R.layout.friend_city_list_item, parent, false));
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull FriendCityViewHolder holder, int position) {
+        holder.friendCityTextView.setText(mCityList.get(position).getCityName());
+        holder.friendCityTextView.setBackgroundColor(getRandomColor());
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/friend/FriendCityRecyclerAdapter.java]: remove this also. Let it be orange color only...

Comment (nikhilbansal97) [reply Android/app/src/main/java/io/github/project_travel_mate/friend/FriendCityRecyclerAdapter.java]: Done. 👍

**Comment by Swati4star** [comment]
Great @nikhilbansal97 👍

**Pull request merged**